lib/node_modules/@stdlib/blas/ext/base/sindex-of/README.md

```diff
@@ -0,0 +1,170 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# sindexOf
+
+> Return the index of a specified search element in a single-precision floating-point strided array.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var sindexOf = require( '@stdlib/blas/ext/base/sindex-of' );
+```
+
+#### sindexOf( N, searchElement, x, strideX\[, offsetX ] )
```

Comment:
@headlessNode This is not correct. Our practice is to **always** have separate APIs. One for providing an offset and another in which the offset is computed. This follows BLAS conventions. I am not sure why you decided to make the API variadic.